app/models/street/StreetEdgeTable.scala

```diff
@@ -199,6 +199,48 @@ object StreetEdgeTable {
     }
   }
 
+
+  /**
+    * Calculates the distance audited today by all users
+    * 
+    * Author: Aroosh Kumar
```

Comment:
Can you remove the author and date from the comments? I think those pieces are not really helpful or necessary, they just kind of take up space. Git/GitHub take care of all the attribution.